Task 1409987: Refresh of .NET article, Install an assembly into the GAC (#10314)
dotnet/docs: docs/framework/app-domains/how-to-install-an-assembly-into-the-gac.md

> [!IMPORTANT]
> You can install only strong-named assemblies into the GAC. For information about how to create a strong-named assembly, see [How to: Sign an assembly with a strong name](how-to-sign-an-assembly-with-a-strong-name.md).

## Windows Installer

[Windows Installer](https://docs.microsoft.com/en-us/windows/desktop/Msi/installation-of-assemblies-to-the-global-assembly-cache), the Windows installation engine, is the recommended way to add assemblies to the global assembly cache. Windows Installer provides reference counting of assemblies in the global assembly cache and other benefits. To create an installer package for Windows Installer, use the [WiX toolset extension for Visual Studio 2017](https://marketplace.visualstudio.com/items?itemName=RobMensching.WixToolsetVisualStudio2017Extension).

## Global assembly cache tool

You can use the [global assembly cache tool (gacutil.exe)](../tools/gacutil-exe-gac-tool.md) to add assemblies to the global assembly cache and to view the contents of the global assembly cache.

   > [!NOTE]
   > *Gacutil.exe* is for development purposes only. Don't use it to install production assemblies into the global assembly cache.

> The syntax for *gacutil.exe* is as follows:

Reviewer: ```suggestion
The syntax for using *gacutil.exe* to install an assembly in the GAC is as follows:
```
I think it's best to make it clear that this is only one of the syntactical variants.

```console
gacutil -i <assembly name>
```

In this command, *\<assembly name>* is the name of the assembly to install in the global assembly cache.

> If *gacutil.exe* isn't in your system path, use the [Developer Command Prompt for Visual Studio](https://docs.microsoft.com/en-us/dotnet/framework/tools/developer-command-prompt-for-vs). 

Reviewer: ```suggestion
If *gacutil.exe* isn't in your system path, use the [Developer Command Prompt for VS *version*](https://docs.microsoft.com/en-us/dotnet/framework/tools/developer-command-prompt-for-vs). 
```

The following example installs an assembly with the file name *hello.dll* into the global assembly cache.

```console
gacutil -i hello.dll
```

> [!NOTE]

> > In earlier versions of the .NET Framework, the *Shfusion.dll* Windows shell extension enabled you to install assemblies by dragging them to File Explorer. Beginning with .NET Framework Version 4, *Shfusion.dll* is obsolete.

Reviewer: ```suggestion
> In earlier versions of the .NET Framework, the *Shfusion.dll* Windows shell extension let you install assemblies by dragging them to File Explorer. Beginning with .NET Framework Version 4, *Shfusion.dll* is obsolete.
```
"Version" isn't necessary -- .NET Framework 4 is adequate.
